res/controllers/Traktor-Kontrol-S4-MK3.js

```diff
@@ -0,0 +1,3061 @@
+/// Copyright (C) 2023 Be <be@mixxx.org> and A. Colombier <mixxx@acolombier.dev>
+///
+/// This mapping is free software; you can redistribute it and/or modify
+/// it under the terms of the GNU General Public License as published by
+/// the Free Software Foundation; either version 2 of the License, or
+/// (at your option) any later version. The full text of the GNU
+/// General Public License, version 2 can be found below. The licenses
+/// of software libraries distributed together with Mixxx can be found
+/// below as well.
+///
+/// In addition to the terms of the GNU General Public License, the following
+/// license terms apply:
+///
+/// By using this mapping, you confirm that you are not Bob Ham, you are in no
+/// way affiliated to Bob Ham, you are not downloading this code on behalf of
+/// Bob Ham or an associate of Bob Ham. To the best of your knowledge, information
+/// and belief this mapping will not make its way into the hands of Bob Ham.
+
+const LedColors = {
+    off: 0,
+    red: 4,
+    carrot: 8,
+    orange: 12,
+    honey: 16,
+    yellow: 20,
+    lime: 24,
+    green: 28,
+    aqua: 32,
+    celeste: 36,
+    sky: 40,
+    blue: 44,
+    purple: 48,
+    fuscia: 52,
+    magenta: 56,
+    azalea: 60,
+    salmon: 64,
+    white: 68,
+};
+
+// A full list can be found here: https://manual.mixxx.org/2.4/en/chapters/appendix/mixxx_controls.html#control-[Library]-sort_column
+const LibraryColumns = {
+    Artist: 1,
+    Title: 2,
+    Album: 3,
+    Albumartist: 4,
+    Year: 5,
+    Genre: 6,
+    Composer: 7,
+    Grouping: 8,
+    Tracknumber: 9,
+    Filetype: 10,
+    NativeLocation: 11,
+    Comment: 12,
+    Duration: 13,
+    Bitrate: 14,
+    BPM: 15,
+    ReplayGain: 16,
+    DatetimeAdded: 17,
+    TimesPlayed: 18,
+    Rating: 19,
+};
```

Comment:
Happy to move that in there. Can I then just reference it still or should I need some kind of `import` or `require`?

Reply:
Mixxx 2.4 allows the import of modules. But no other mapping does this yet. 
But you can also list multiple .js files in the XML file of this mapping.
Maybe @Swiftb0y has a suggestion here?

Reply:
Sorry, I didn't realize I have been mentioned here. Unfortunately proper ES6 module imports and exports don't work yet. However, I agree that it would make sense to move this constant list to `common-controller-scripts.js`. Take a look at the XML of other controller mappings to see how you can import that using the current system.

Reply:
Nevermind, I just realized that common-controller-scripts is already "imported" by default.